openstack/datadog_checks/openstack/data/conf.yaml.example

```diff
@@ -1,91 +1,118 @@
+  #    - <NETWORK_NAME>
+  #    - <NETWORK_PREFIX>.*
+
+  ## @param ssl_verify - boolean - optional - default: true
+  ## Whether to enable SSL certificate verification for HTTP requests. You may
+  ## need to set it to false when using self-signed certs.
+  #
+  #  ssl_verify: true
+
+  ## @param use_agent_proxy - boolean - optional - default: false
+  ## Whether the dd-agent proxy should also be used for Openstack API requests (if set).
+  #
+  #  use_agent_proxy: false
 
 instances:
-    - name: instance_1 # A required unique identifier for this instance
-
-      # The authorization scope that will be used to request a token from Identity API v3
-      # The auth scope must resolve to 1 of the following structures:
-      # {'project': {'name': 'my_project', 'domain': 'my_domain} OR {'project': {'id': 'my_project_id'}}
-      #
-      # ALPHA: DO NOT ADD an auth_scope if you wish to have unscoped access
-      #        Simply add the relevant user credentials.
-      #
-      auth_scope:
-          project:
-              id: my_project_id
-
-          # Alternately
-
-          # project:
-          #     name: my_project_name
-          #     domain:
-          #         id: default
-
-      # All projects in a domain can be collected by enabling collect_all_projects 
-      # Note: `domain` field must be set in project
-      #
-      # collect_all_projects: true
-
-
-      # User credentials
-      # Password authentication is the only auth method supported right now
-      # User expects username, password, and user domain id
-      # `user` should resolve to a structure like {'password': 'my_password', 'name': 'my_name', 'domain': {'id': 'my_domain_id'}}
-      user:
-          password: my_password
-          name: datadog
-          domain:
-              id: default
-
-      # In some cases, the nova url is returned without the tenant id suffixed
-      # e.g. http://172.0.0.1:8774 rather than http://172.0.0.1:8774/<tenant_id>
-      # Setting append_tenant_id to true manually adds this suffix for downstream requests
-      # append_tenant_id: false
-
-      # Please note that server metrics override the host tag and thus do not get
-      # the host-level tags specified in datadog.(conf/yaml)
-      # To apply tags to all openstack metrics, you can use the following "tags" option 
-      #
-      # Set (optional) custom tags for each metric
-      # tags:
-      #   - optional:tag1
 
+  ## @param name - string - required
+  ## Unique identifier for this instance.
+  #
+  - name: <INSTANCE_NAME>
+
+  ## @param user - custom object - required
+  ## User credentials
+  ## Password authentication is the only auth method supported.
+  ## `user` object expects the parameter `username`, `password`, and `user.domain.id`.
+  ##
+  ## `user` should resolve to a structure like:
+  ##
+  ##  {'password': '<PASSWORD>', 'name': '<USERNAME>', 'domain': {'id': '<DOMAINE_ID>'}}
+  #
+    user:
+      password: <PASSWORD>
+      name: datadog
```

Comment:
idk for this one, since in the installation process we ask the customer to create a user named `datadog` for the access. Hence leaving it as the "default" valued maid sens.. but we can indeed switch it back to the <USERNAME> placeholder.

Reply:
OK makes sense